media-video/webcamoid/webcamoid-8.1.0.ebuild

```diff
@@ -0,0 +1,99 @@
+# Copyright 1999-2017 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+PLOCALES="ca de el es et fr gl it ja kab ko nl pt ru uk zh_CN zh_TW"
+
+inherit l10n qmake-utils
+
+DESCRIPTION="A full featured webcam capture application"
+HOMEPAGE="https://webcamoid.github.io"
+SRC_URI="https://github.com/webcamoid/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+KEYWORDS="~amd64 ~x86"
+LICENSE="GPL-3"
+SLOT="0"
+
+IUSE_AVKYS=( alsa coreaudio ffmpeg gstreamer jack libuvc oss pulseaudio qtaudio v4lutils videoeffects )
+IUSE="${IUSE_AVKYS[@]} debug doc headers v4l"
+
+REQUIRED_USE="
+	v4lutils? ( v4l )
+"
+
+RDEPEND="
+	dev-qt/qtconcurrent:5
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtquickcontrols:5
+	dev-qt/qtsvg:5
+	dev-qt/qtwidgets:5
+	ffmpeg? ( >=media-video/ffmpeg-3.1.0:= )
```

Comment:
No chance for `media-video/libav:=`? Usually we do this dance: `ffmpeg? ( libav? ( ... ) !libav? ( ... ) )`

Reply:
Could you tell me which minimal version of `libav` to specify?

Reply:
ffmpeg was first introduced a year and a half ago. Most likely, anything we have in the tree is going to be good enough.

If upstream doesn't give you a specific version, then any version should work.

Reply:
Is it really need make possible to set both flags?

Reply:
That is being nice to libav users, yes. ffmpeg-only should be reserved in cases were it is known to absolutely do not work at all. It does not mean you have to test with a libav image, though.